lua/pendulum/init.lua

```diff
@@ -10,12 +10,21 @@ local default_opts = {
     timer_len = 120,
     gen_reports = true,
     top_n = 5,
+    report_excludes = {
+        filetype = {},
+        branch = {},
+        directory = {},
+        project = {},
+        file = {},
+        group = {},
```

Comment:
I think putting the section exclude (group) alongside the more specific ones is a bit confusing.
I think separating them into two different tables would be better (maybe called "report_excludes" and "report_section_excludes"?)

Reply:
That's easy enough